docs/guides/tools-reference/file-transfer/comparing-data-transfer-utilities/index.md

```diff
@@ -0,0 +1,128 @@
+authors: ["Leon Yen"]
+contributors: ["Leon Yen"]
+published: 2024-10-03
+keywords: ['data transfer utility','data transfer','file transfer']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+external_resources:
+- '[Introduction to Rsync](/docs/guides/introduction-to-rsync/)'
+- '[Transfer Files With the scp Command on Linux](/docs/guides/how-to-use-scp/)'
+- '[Transfer Files with SFTP](/docs/guides/sftp-linux/)'
+- '[What Is FTP and How Does It Work?](/docs/guides/what-is-ftp/)'
+- '[Transfer Files with FileZilla](/docs/guides/filezilla/)'
+---
+
+Technology professionals and end users rely on a myriad of utilities for moving data between locations. This guide discusses several of these data transfer utilities -- rclone, rsync, MiniIO, SCP, SFTP, and FTP/SFTP clients -- and details each of their core features, use cases, key considerations, strengths, and limitations.
+
+## What Is a Data Transfer Utility?
+
+A data transfer utility’s primary purpose is to facilitate the movement of data between different systems/devices, networks, and IT environments. At its core, a data transfer utility must transmit data from origin to destination efficiently and securely, with minimal disruption and errors. Depending on the use case, different data transfer tools may work better than others.
+
+Common data transfer scenarios range from simple file transfers and sharing to full-scale system backups, real-time data replication, and data migrations between different on-premise or cloud environments. Each use case often benefits from a different type of data transfer utility.
+
+## Transfer Considerations
+
+The following are some general transfer considerations to keep in mind when selecting a data transfer utility for a particular use case or scenario.
+
+### Egress and Ingress Cost
+
+Copying a single file from a desktop client to a cloud server calls for a different data transfer approach than migrating large volumes of data at scale between enterprise IT environments. When evaluating data transfer utilities, you should consider relative egress and ingress costs and the volume of data being transferred.
+
+### Data Storage Size and Type
+
+Along with how much data is being transferred, the kind of data and the destination are equally important (e.g., your data’s source and target destination, their respective file systems, and how much storage is required once the data is at rest).
+
+For example, most end users only deal with traditionally structured file systems when using their local environment, with data organized hierarchically into folders. In contrast, block storage breaks up files into separate blocks and stores them separately, allowing for significant performance benefits. Object storage saves files in a non-hierarchical, self-contained “flat” format. This allows for increased scalability and sizable reductions in cost and complexity.
```

Comment:
I don't think block storage/block devices stand in-contrast with file systems; file systems are implemented on block devices. So, I think this paragraph could use a bit of a rewrite, which I'll take a swing at in a bit

Reply:
https://en.wikipedia.org/wiki/Block-level_storage
https://en.wikipedia.org/wiki/Block_(data_storage)